public/index.html

```diff
@@ -9,7 +9,10 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <!-- fontawesome is the site used to import logos for javascript -->
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <script src="https://kit.fontawesome.com/e81e313b45.js" crossorigin="anonymous"></script>
```

Comment:
For these you can use https://www.npmjs.com/package/react-icons, it has support for font-awesome